src/neo/SmartContract/InteropService.Oracle.cs

```diff
@@ -0,0 +1,36 @@
+using Neo.Oracle.Protocols.HTTPS;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Oracle
+        {
+            public static readonly uint Neo_Oracle_HTTPS11_Get = Register("Oracle.HTTPS11.Get", Oracle_HTTPS11_Get, 0, TriggerType.Application, CallFlags.None);
+
+            private static bool Oracle_HTTPS11_Get(ApplicationEngine engine)
+            {
```

Comment:
Even through it's fine, but it's not very general. As we'll also need to support both `https` and `neofs` protocol. 
https://github.com/neo-project/neo/issues/1270#issuecomment-558153166

I prefer to rename it `Oracle_Get`, and we can parse the URI.

Reply:
Agree. We can rename the SYSCALL to `System.Oracle.Get`.

Reply:
Without HTTPS?

Reply:
`System.Oracle.HTTPS.Get` ?

Reply:
We can detect the protocol from the URI.